app/facades/player_facade.rb

```diff
@@ -10,6 +10,17 @@ def self.class_info(name)
     DndClass.new(raw_data, class_description)
   end
 
+  def self.find_by_id(player_id)
+    player = Player.where(id: player_id).first
```

Comment:
should be able to use `Player.find(params[:id])` here rather than `.where` - that way you don't need to call `.first` after